src/EditorFeatures/CSharpTest2/Recommendations/AbstractKeywordRecommenderTests.cs

```diff
@@ -256,6 +256,13 @@ public async Task TestAfterPublic()
 @"public $$");
         }
 
+        [Fact]
+        public async Task TestAfterFile()
+        {
+            await VerifyKeywordAsync(SourceCodeKind.Regular,
+@"file $$");
+        }
```

Comment:
I'm thinking of a test case like this:

```csharp

file $$

class C
{
}
```

Does it work currently? Is it hard to support?

Reply:
added in new test and passed! Thanks Youssef!